_setup/software_setup.md

```diff
@@ -84,22 +98,4 @@ We've created a **code repository** that will get your board ready to program an
 
 1. Type `git clone https://github.com/byu-cpe/y-badge-code.git` and press `Enter`
 
-1. Type `code y-badge-code` and press `Enter`
-
-## Source Code Layout
```

Comment:
@KeenanRileyFaulkner Did you mean to delete all this?

Reply:
Yes, moved to _Coding Intro_ because when my wife was going through the docs she thought that was a more intuitive place. If you don't like it, I can change it back.